fhir_2_ada-r4/zibs2020/payload/0.0/all_zibs.xsl

```diff
@@ -26,31 +26,31 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
     <xsl:import href="contextContactEpisodeOfCare.xsl"/>
     <xsl:import href="ext-CopyIndicator.xsl"/>
     <xsl:import href="ext-MedicationAgreementPeriodOfUseCondition.xsl"/>
-    <xsl:import href="ext-TimeInterval-period.xsl"/>
     <xsl:import href="ext-TimeInterval-Duration.xsl"/>
+    <xsl:import href="ext-TimeInterval-period.xsl"/>
```

Comment:
ext-TimeInterval-Period (?)
Or even better: ext-TimeInterval.Period (in line with the name of the extension)

Reply:
Good question but out of scope for this ticket. Created new ticket: MP-1189